foyer/tests/test_plugin.py

```diff
@@ -6,9 +6,11 @@ def test_basic_import():
     assert 'forcefields' in dir(foyer)
 
 
-@pytest.mark.parametrize('ff_name', ['OPLSAA', 'TRAPPE_UA'])
-def test_forcefields_exist(ff_name):
-    ff_name in dir(foyer.forcefields)
+def test_loading_forcefields():
+    #funcs = [func for func in dir(foyer.forcefields) if 'load' in func and '__' not in func]
+    for func in dir(foyer.forcefields):
+        if 'load_' in func and '__' not in func:
+            eval('foyer.forcefields.' + func)()
```

Comment:
Im ok with this, but just want to note that eventually this might be a possible issue. However, I cannot think of a better way to handle this.

Reply:
I agree, `eval` is dangerous - could you raise an issue so we don't forget?